### PR TITLE
AB#87742 Site - Landing Page and Name views

### PR DESCRIPTION
**Pull request by Carq**
image

image

### CODE REVIEW

File: src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                <li>Building for a Healthy Life criteria</li>
+                <li>land and tendering status</li>
+                <li>site details</li>
+                <li>Modern Methods of Construction (MMC)</li>
+                <li>procurement</li>
+            </ul>
+            <gds-p>If your application concerns existing properties rather than land-led development, this section is still relevant to you. Complete this section in relation to the properties and where we use 'site' or 'land', substitute in 'properties'.</gds-p>
+            <gds-p>We will also ask you to tell us if you’re using Modern Methods of Construction (MMC). If your site includes a mix of MMC and traditional build homes, you can indicate that here. We will ask for more details when you tell us about the homes on your site.</gds-p>
+        </div>
+        <button role="navigation" id="start-now-button" class="govuk-button govuk-button--start" data-module="govuk-button" name="action" type="submit" value="Continue">
```
Comment (Antek012) [inline src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml:26]: use gds-button-start

**Review by Antek012 — APPROVED** [review]

File: src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+@using HE.Investment.AHP.WWW.Views.Site
+@using HE.Investments.Common.WWW.Components.FormHeader
+@using HE.Investment.AHP.Contract.Site
+
+<gds-div-grid-column-two-thirds>
+    <form asp-controller="Site" asp-action="Start"  asp-method="post" novalidate>
```
Comment (jjankowski87) [inline src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml:6]: Do we need form for static view? Can we use link with redirect?

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml (hunk 2 above)
Comment (Antek012) [reply src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml:6]: +1

File: src/HE.Investment.AHP.WWW/Views/Site/Name.cshtml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <gds-back-link href=@Url.Action("Back", "Site", new { currentPage = SiteWorkflowState.Name }) class="govuk-!-margin-bottom-8"></gds-back-link>
+    @await Html.PartialAsync("~/Partials/Errors/_ErrorSummaryPartial.cshtml")
+    <form asp-controller="Site" asp-action="Name" asp-method="post" novalidate>
+        <vc:form-header title="@SitePageTitles.SiteName" />
+        <gds-field-set role="group">
+            <legend>
+                <gds-p >Each site needs a unique name.</gds-p>
+            </legend>
+
+            <gds-input asp-hiddenlabeltext="Name"
```
[the remark this answers is not in the document]
Comment (Carq) [reply src/HE.Investment.AHP.WWW/Views/Site/Name.cshtml:22]: It is displayed.

File: src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml (hunk 1 above)
Comment (Carq) [reply src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml:26]: I will do this in the next CR.

File: src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml (hunk 2 above)
Comment (Carq) [reply src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml:6]: I will do this in the next CR.

**Pull request merged**